cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -51,26 +57,18 @@ data:
       pos_file /var/log/gcp-containers.log.pos
       tag reform.*
       read_from_head true
-      format none
+      format multi_format
```

Comment:
NOTE:  This entire `format` change is to revert the changes we made in https://github.com/kubernetes/kubernetes/pull/59128/files#diff-dec77c261fefaa453d67b0c26b4b07c2 to adapt to Fluentd `v0.14` syntax. As we downgrade back to `v0.12` in this PR, the syntax needs to be reverted as well.